pkg/ipamd/datastore/data_store_test.go

```diff
@@ -925,8 +925,8 @@ func TestGetIPStatsV4(t *testing.T) {
 		*ds.GetIPStats("4"),
 	)
 
-	// wait 30s (cooldown period)
-	time.Sleep(30 * time.Second)
+	// wait number of seconds configured in IP_COOLING_PERIOD (cooldown period)
```

Comment:
Can we add a test case where we actually set `IP_COOLING_PERIOD`?

Reply:
yes, it would help though if you could point me to a similar test case.